fem/transfer.cpp

```diff
@@ -679,11 +1835,55 @@ void L2ProjectionGridTransfer::L2ProjectionH1Space::Mult(
    }
 
    SetFromTDofs(fes_lor, Y, y);
+
+   if (verify_solution)
+   {
+      Vector y_temp(y.Size());
+      DeviceMult(x, y_temp);
+      y_temp -= y;
+      real_t error = y_temp.Norml2();
+      if (error > ho_lor_tol)
+      {
+         MFEM_VERIFY(false, "Mult difference too high = "<<error);
+      }
+   }
+}
+
+void L2ProjectionGridTransfer::L2ProjectionH1Space::DeviceMult(
+   const Vector& x, Vector& y) const
```

Comment:
Also nearly identical to `L2ProjectionH1Space::Mult`, could get rid of this function if you don't need verification and change construction to put `R_vea` into `R`.